btree/btree.c

```diff
@@ -7879,7 +7841,8 @@ struct m0_ut_suite btree_ut = {
 	.ts_init = ut_btree_suite_init,
 	.ts_fini = ut_btree_suite_fini,
 	.ts_tests = {
-		{"basic_tree_op",                   ut_basic_tree_oper},
+		{"basic_tree_op_cp",                ut_basic_tree_oper_cp},
+		//{"basic_tree_op_icp",               ut_basic_tree_oper_icp},
```

Comment:
Any reason behind disabling this test ?

Reply:
This test is the incorrect path test.
At the moment, one of the scenario i.e. calling destroy after tree has been already destroyed is not gracefully failing.
Hence, the test has been disabled.

Reply:
Is it going to be part of any other ticket?